safe/definitions/__init__.py

```diff
@@ -27,3 +28,4 @@
 from safe.definitions.units import *
 from safe.definitions.versions import *
 from safe.definitions.reports import *
+from safe.definitions.post_processors import *
```

Comment:
__[flake8]__
* [F403] 'from safe.definitions.post_processors import *' used; unable to detect undefined names
* [F811] redefinition of unused 'safe.definitions.post_processors.*' from line 27
* [F401] 'safe.definitions.post_processors.*' imported but unused

<!-- Comment by [SideCI](https://sideci.com) -->